owncloudApp/src/androidTest/java/com/owncloud/android/settings/security/PassCodeActivityTest.kt

```diff
@@ -128,14 +124,28 @@ class PassCodeActivityTest {
         R.id.explanation.isDisplayed(false)
 
         // Check if required amount of input fields are actually displayed
-        with(R.id.passCodeTxtLayout) {
+        with(R.id.layout_code) {
```

Comment:
Just a note, when you change the Id of elements of the view already made, warn the rest of the team, sometimes certain tests can be broken 😉

Reply:
Okay, thank you